src/main/java/mods/railcraft/world/level/block/TrackBlock.java

```diff
@@ -164,10 +164,10 @@ public boolean canMakeSlopes(BlockState state, IBlockReader world, BlockPos pos)
     return TrackSupportTools.isSupportedDirectly(world, pos);
   }
 
-  @Override
+  @Override // this is about to be depricated, we need to do our own "canSurvive"
```

Comment:
This won't be deprecated, `@Deprecated` annotations on `Block` methods mean you should call the block state variant instead.